### PR TITLE
Gce report fix

### PR DESCRIPTION
**Pull request by confunguido**
Merging both reports into one. I'm also removing unused parameters related to the reports.

### CODE REVIEW

File: scripts/plot_outputs.R [hunk 1]
```diff
@@ -9,11 +9,6 @@ ggplot2::theme_set(ggplot2::theme_classic())
...
 ## ===============================#
 
-demographic_report <- readr::read_csv(file.path(
-  "output",
-  "person_demographics_count.csv"
-))
-
 infectious_report <- readr::read_csv(file.path(
   "output",
   "person_infectious_count.csv"
```
Comment (ChiragKumar9) [inline scripts/plot_outputs.R:14]: The report here is called the `person_infectious_count` whereas in `main` it's called `person_property_count`. Please update the report name here.

**Review by ChiragKumar9 — CHANGES_REQUESTED** [review]

**Review by KOVALW — APPROVED** [review]
LGTM apart from Chirag's catch on the `person_property_count` variable name

File: scripts/plot_outputs.R (hunk 1 above)
Comment (confunguido) [reply scripts/plot_outputs.R:14]: I'm very curious of how hat even happened. We plotted something and "person_infectious_count.csv" does not exist on my computer.

**Review by ChiragKumar9 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**